worldeditadditions_farwand/lib/cloudwand.lua

```diff
@@ -1,52 +1,22 @@
-local function add_point(name, pos)
```

Comment:
Oh, good thinking. If we move this to the main `worldeditadditions` mod, then it can become common API logic across your new `//scloud` and my existing wand.

Reply:
Yah it really makes things easier